composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "guzzlehttp/oauth-subscriber",
+    "name": "aerendir/oauth-subscriber",
```

Comment:
you can download your fork without publishing it to packagist, infact it's better to do that